tests/installation/scc_registration.pm

```diff
@@ -21,7 +21,10 @@ use registration;
 
 sub run() {
     if (!get_var("HDD_SCC_REGISTERED")) {
-        assert_screen("scc-registration", 100);
+        if (!check_screen 'scc-registration', 210) {
+            record_soft_failure 'bsc#990254';
+            assert_screen('scc-registration', 100);
+        }
```

Comment:
that does not make sense to me. In this case you would have three cases

* *passed* when scc-registration found faster than 300
* *softfailed* when scc-registration found between 300-310 seconds
* *failed* when scc-registration not found in 311 seconds